src/test_workflow/benchmark_test/benchmark_test_suite.py

```diff
@@ -98,7 +98,7 @@ def convert(self) -> None:
             file_path = glob.glob(os.path.join(str(work_dir.path), "*", "test_execution.json"))
             final_results_file = glob.glob(os.path.join(str(work_dir.path), "final_result.md"))
             shutil.copy(file_path[0], os.path.join(os.getcwd(), f"test_execution_{self.args.stack_suffix}.json"))
```

Comment:
Will we need to archive the json file in future for this usecase, if yes can you update the copy path for test_execution.json also